meetings/notes/sig-meeting-20221115.md

```diff
@@ -0,0 +1,57 @@
+## Meeting Details
+
+- **Date/Time:** [November 15th, 2022 @ XX:XXpm UTC / XX:XXpm ET](https://lists.o3de.org/g/o3de-calendar/viewevent?repeatid=39350&eventid=1557573&calstart=2022-11-15)
+- **Location:** [Discord SIG-Network Voice Room](https://discord.gg/62nq7HP5mP)
+- **Moderator:** Pip Potter (lmbr-pip)
+- **Note Taker** Pip Potter (lmbr-pip)
+
+[Agenda Issue](https://github.com/o3de/sig-network/issues/79)
+
+## SIG Updates
+
+### General SIG announcements
+AWS is looking to contribute a [major update](https://github.com/o3de/o3de/issues/13198) to the 3rd person Multiplayer Sample
+
+[O3DCon 2022](https://events.linuxfoundation.org/o3dcon/) talks now on [YouTube](https://www.youtube.com/playlist?list=PLCQwFpnHSZQgzCpMmbxruFkWr3d73ZfEJ). Of interest to SIG:
+    * [Building Multiplayer Content for Large Worlds](https://www.youtube.com/playlist?list=PLCQwFpnHSZQgzCpMmbxruFkWr3d73ZfEJ)
+    * [Scale Testing Multiplayer Games on AWS](https://www.youtube.com/playlist?list=PLCQwFpnHSZQgzCpMmbxruFkWr3d73ZfEJ)
```

Comment:
all the talk links in this list appear to be the playlist, not the individual talk links

Reply:
Good catch, I started each talk and just copied the URL, I guess Youtube was tracking this via the playlist, will update.